CatAnalyzer/plugins/TtbarDiLeptonAnalyzer.cc

```diff
@@ -318,9 +310,11 @@ void TtbarDiLeptonAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSe
   if      ( b_channel == CH_ELEL ) iEvent.getByToken(trigTokenELEL_, trigHandle);
   else if ( b_channel == CH_MUMU ) iEvent.getByToken(trigTokenMUMU_, trigHandle);
   else if ( b_channel == CH_MUEL ) iEvent.getByToken(trigTokenMUEL_, trigHandle);
-  if ( !trigHandle.isValid() ) return;
+  if ( !trigHandle.isValid() ){
+    ttree_->Fill();
+	return;
+  }
   b_tri = *trigHandle;
-  //if (!tri) return;
```

Comment:
I just checked 'channel != 0 && tri == -9' cut returns 0 events for every mc&data with this code. so I think we can get rid of 'trigHandle.isValid()' test.(?)